Sources/FluentUI_iOS/Core/Theme/Tokens/LinearGradientInfo.swift

```diff
@@ -3,20 +3,19 @@
 //  Licensed under the MIT License.
 //
 
-import CoreGraphics
 import Foundation
 import SwiftUI
 
 /// Represents a linear gradient as used by FluentUI.
-@objc public class LinearGradientInfo: NSObject {
+public class LinearGradientInfo: NSObject {
```

Comment:
any reason to drop objc?

Reply:
Ah yeah I had a few reasons:
1. It currently doesn't have any `@objc` methods or properties, so it couldn't be created or used anyway.
2. The `@objc` was missing the `MSF` prefix, which I would rather we include whenever possible.
3. We can't make the new initializer `@objc` anyway, since it uses SwiftUI types now (which can't be expressed in Objective-C).